src/OrchardCore.Modules/OrchardCore.Workflows/Views/Items/WorkflowFaultEvent.Fields.Thumbnail.cshtml

```diff
@@ -0,0 +1,2 @@
+<h4 class="card-title"><i class="fa fa-exclamation-circle"></i>@T["Workflow Fault Event"]</h4>
```

Comment:
Are we duplicate the titles in other thumbnails?

Reply:
How about this one instead?

![image](https://user-images.githubusercontent.com/15613121/184533179-8e359684-8563-4e82-bbd7-05ec1124c883.png)

Reply:
I mean "Workflow Fault Event"

Reply:
updated to 
`Trigger and capture exception information when exception occurs in any other workflow.`